element_array_ephys/export/nwb/nwb.py

```diff
@@ -0,0 +1,637 @@
+import os
+import decimal
+import json
+import numpy as np
+import pynwb
+import datajoint as dj
+from element_interface.utils import find_full_path
+from hdmf.backends.hdf5 import H5DataIO
+from hdmf.data_utils import GenericDataChunkIterator
+from nwb_conversion_tools.tools.nwb_helpers import get_module
+from nwb_conversion_tools.tools.spikeinterface.spikeinterfacerecordingdatachunkiterator import (
+    SpikeInterfaceRecordingDataChunkIterator
+)
+from spikeinterface import extractors
+from tqdm import tqdm
+import warnings
+from ... import probe, ephys_acute
+
+assert probe.schema.is_activated(), 'probe not yet activated'
+
+for ephys in (ephys_acute,):
+    if ephys.schema.is_activated():
+        break
+else:
+    raise AssertionError('ephys not yet activated')
+
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, decimal.Decimal):
+            return str(o)
+        return super(DecimalEncoder, self).default(o)
+
+
+class LFPDataChunkIterator(GenericDataChunkIterator):
```

Comment:
Thanks @bendichter.  This will be great to have.  I am unfortunately not too familiar with this implementation so I won't be able to provide a thorough review on this `class`.